src/main/java/seedu/address/logic/commands/AddNoteCommand.java

```diff
@@ -77,9 +77,29 @@ public CommandResult execute(Model model) throws CommandException {
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
 
+        Person person = FindCommand.getFoundPerson();
+        if (person != null && isNoteChanged(person, note)) {
+            model.updateFilteredPersonList(new IdentityCardNumberMatchesPredicate(person.getIdentityCardNumber()));
+            model.setDisplayNoteAsFirstFilteredPerson();
```

Comment:
@jiahui0309 this also seems like a hackish way to try and get to the first person. It might be better to move the manipulation of the displayed note to person. Maybe `model` can have a method `setDisplayNoteAsPerson(Person)` that sets the display note to be the Person of our choice?